src/scanoss/cli.py

```
@@ -25,9 +25,12 @@
 import argparse
 import os
 import sys
+from array import array
```

Ruff (0.8.2)

`array.array` imported but unused

Remove unused import: `array.array`

(F401)